### ISSUE TITLE
[CI] CI tests are broken on master

### ISSUE THREAD

**Issue by ramrodo** [layer5io/meshery#2157]
**Description**
CI tests against master are broken:

- golangci-lint:
https://github.com/layer5io/meshery/runs/1626761249?check_suite_focus=true
```
  Error: Error return value of `encoder.Encode` is not checked (errcheck)
  Error: Error return value of `encoder.Encode` is not checked (errcheck)
  Error: don't use ALL_CAPS in Go names; use CamelCase (golint)
```

- UI end-to-end tests
https://github.com/layer5io/meshery/runs/1626774159?check_suite_focus=true 
```
ModuleNotFoundError: Module not found:
Error: Can't resolve '@paciolan/remote-component' in '/home/runner/work/meshery/meshery/ui/components'
```

**Expected Behavior**
CI Tests against master should always pass

**To Reproduce**
1. Create a PR against master
2. See the errors on CI

**Additional Context**

PRs related:
https://github.com/layer5io/meshery/pull/2145

**Comment by ramrodo** [comment]
@utkarsh-pro could you check these issues, please? I'm not too familiar with React Remote Components. Also, check the golangci-lint errros.

**Comment by utkarsh-pro** [comment]
@ramrodo golangci-lint errors have already been addressed and will be fixed soon. The other error is not actually related to remote components, it is a node_modules cache issue. I'll try to invalidate that cache somehow.

**Comment by leecalcote** [comment]
Gents, @kushthedude offered the following counsel:

> - purge the current cache and reinstate node_modules.
> - change the restore and hit keys to get a fresh cache copy.

Reference: https://meshery.slack.com/archives/C011VPREG2Z/p1609274372165300?thread_ts=1609269676.165100&cid=C011VPREG2Z

**Comment by ramrodo** [comment]
closed by #2155 and #2192
